CONTRIBUTING.md

```diff
@@ -61,9 +61,48 @@ If you want to verify the changelog entries as described by a range of Git revis
 
     ./unreleased.sh master..
 
+## Writing a good changelog entry
+
+Writing good changelog entries is **important**: as a developer, it gives visibility on your contribution; as a user, it makes clear what is new, what changes and two deal with those, making the product more accessible and your work more meaningful.
+
+The raw changelog is used to compile a meaningful summary of changes across releases. This happens some time after the PR has been merged and the person taking the responsability of summarizing new user-facing features must be in the position of easily understand the nature of the change and report it. The ideal changelog entry can be more or less incorporated verbatim in the release notes.
+
+Here are a few practical tips
+
+* the first term to appear should be the affected component -- [here's a list](#list-of-components-for-changelog-entries)
+* write as many as necessary changelog entries as necessary
+* don't be _too_ succinct: a single entry does **not have to** fit on a single line
+* on the other end, if the size grows beyond 5-6 lines, rather add a link to a relevant documentation or issue with more details
+* the ultimate target are end users: focus on the impact on them, tell them what's new or how to deal with a change
+
+### List of components for changelog entries
```

Comment:
I notice that the Ledger API Server and kvutils don'tt seem to fall into any of these. I've been using "Ledger Integration Kit" for the latter for a while. Should they be in the list, do they go somewhere else that I'm not seeing, or should they not be included at all?

Reply:
We bundle up all the changes that are only relevant for integrations but not for DAML users under “integration kit” atm.

Reply:
The list should be understood more as a suggestion than a strict rule: try to fit in one of these if possible, but if it doesn't fit, you should feel free to make one up. Better yet: add to the list.

Reply:
@bame-da Your take?

Reply:
@bame-da More precise question: do you think we should add items here? If yes, what should those be?

Reply:
Fair enough, I’ve made a suggestion with the components that we use so far (including bundling stuff that we currently bundle and removing stuff that is not an SDK component (maven plugin, test library)) and changed the spellings that we used so far.